quartz/plugins/transformers/frontmatter.ts

```diff
@@ -91,6 +103,9 @@ declare module "vfile" {
     } & Partial<{
         tags: string[]
         aliases: string[]
+        modified: string
+        created: string
+        published: string
```

Comment:
either we should coerce these values with to string or change the definition[^1] to

```suggestion
        published: string | number
```

because dates might potentially be numbers. e.g. with the frontmatter:

```yaml
modified: 2024-01-01
created: "2024"
published: 2024
```

all three date properties will display in obsidian as `01/01/2024`[^2] but the first two are strings and the last is a number type

[^1]: `.toString()` is probably the way to go. since we're opinionated about the properties anyways
[^2]: quartz currently interprets number types as seconds-since-epoch so `published: 2024` will display as `1970` in quartz. this is fixed in #1646

Reply:
o think we should keep it as string